### PR TITLE
fix: cfs speaker edit properly after cfs is closed 

### LINKED ISSUE #5341: Speaker Edit shows error "call for speaker not issued yet" if there is no CfS
If a user edits and saves a speaker on an event with an deactivated call for speakers an error message appears "call for speaker not issued yet". The form also moves back to the event instead of the speaker page after editing.

Compare: https://eventyay.com/e/1f720482/cfs/speaker/4885/edit

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5341

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@maze-runnar Does this work without changes in the server?

**Comment by maze-runnar** [comment]
> @maze-runnar Does this work without changes in the server?

yes, it is working.

File: app/routes/public/cfs/index.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ export default class IndexRoute extends Route {
      - CFS is private and a valid hash is entered
     */
     if (!speakersCall.announcement) {
-      this.notify.error(this.l10n.t('Call For Speakers has not been issued yet.'));
-      this.transitionTo('public', eventDetails.identifier);
+      // this.notify.error(this.l10n.t('Call For Speakers has not been issued yet.'));
```
Comment (iamareebjamal) [inline app/routes/public/cfs/index.js:36]:
```suggestion
```

**Review by iamareebjamal — APPROVED** [review]

**Comment by iamareebjamal** [comment]
Please don't comment code. Code should never be commented out, it should be deleted. When someone dies, we cremate or bury them, we don't keep them decorated in our home. So why should dead code be treated better?

**Comment by maze-runnar** [comment]
>  When someone dies, we cremate or bury them, we don't keep them decorated in our home. So why should dead code be treated better?

ok :expressionless:

**Pull request merged**